### ISSUE TITLE
When configured as slave, allow processing of multiple jobs simultaneously

### ISSUE THREAD

**Issue by thombergs** [reflectoring/coderadar#130]
There are a couple jobs that are stored into the database by *Trigger classes. These jobs are read from the database and processed by all server nodes that are configured with `coderadar.slave=true`.

Currently, each slave only processes one job at a time. It should be possible to configure how many jobs should be processed simultaneously to take advantage of all processor resources.

* add an integer configuration parameter `coderadar.slave.maxParallelJobs` (implement ConfigurationParameter)
* extend JobExecutionService to add threading to allow the configured number of jobs to execute in parallel

Hints:
* the jobs are started once you create a project via the [REST API](http://www.reflectoring.io/coderadar/1.0.0-SNAPSHOT/docs/restapi.html#_creating_a_project)
* only AnalyzeCommitJobs can really be run in parallel, since the other jobs are triggered sequentially. These jobs start, once you [configured an analyzing strategy](http://www.reflectoring.io/coderadar/1.0.0-SNAPSHOT/docs/restapi.html#_setting_a_project_s_analyzing_strategy) via REST API.

**Comment by rzpio** [comment]
The job framework is dropped (see #204), so this is no longer needed. If a job framework is added again, the issue can be opened again.
